baseapp/baseapp/activity_log/README.md

```
@@ -0,0 +1,104 @@
+# BaseApp Activity Log
+
+This app exposes an API to view user's activities that are being tracked by the system.
+
+## Installation
+
+### Requirements:
```

Markdownlint (0.35.0)

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)